### PR TITLE
Pull data from Tock api

### PR DESCRIPTION
**Pull request by lauraGgit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,38 +9,32 @@ git clone https://github.com/18F/tock-blocks.git
...
-
 ## Run the tock blocks
 
 `python server.py -p tock-blocks -u firstName.lastName -s YYYY-MM-DD -e YYYY-MM-DD`
 
 It will then print out an array of your major blocks of projects (rounded to the nearest 5%), as well of a list of other projects that are greater than 2.5% of your time.
 
 ### Running the Regular Utilization Report
-1. Follow the download instructions from above using the "complete timecard data with fewer fields" button on the Tock report (if you have excel downloaded, don't save the file as because that will change the csv format and break tock-blocks).
+1. Set TOCK_API_KEY environment variable with `export TOCK_API_KEY=<your_key>`
```
Comment (stvnrlly) [inline README.md:19]: How does one get a Tock API key?

File: README.md [hunk 2]
```diff
@@ -9,38 +9,32 @@ git clone https://github.com/18F/tock-blocks.git
...
 2. `cd tock-blocks`
 3. Make sure it is running in Python 3. Verify by checking `pyenv local`
-4. Running the command ` python server.py -f slim_timecard_bulk.csv`. Months must be the full name with capitalization i.e "September" or "July".
+4. Running the command ` python server.py`.
```
Comment (stvnrlly) [inline README.md:22]: Running this without the timecard file or the users CSV specified (or in the folder) raises an error.

File: tock_blocks.py [hunk 3]
```diff
@@ -118,6 +118,8 @@ class color:
 
 # Function for an array of person
 def block_by_array(users, entries, start, end, nice_display, exclude_leave):
+    if args.file is not None:
+        time_entries = tock_blocks.read_CSV_to_list(args.file)
```
Comment (stvnrlly) [inline tock_blocks.py:122]: Will this still work if a JSON file is used?

**Review by stvnrlly — APPROVED** [review]
It looks like there are still a few holdover from CSV-world that could use cleanup later, but this seems pretty straightforward.

I'd recommend explaining the setup in more detail, since the process of getting and setting an API key may be new to some users.

File: tock_blocks.py (hunk 3 above)
Comment (lauraGgit) [reply tock_blocks.py:122]: ah good point, let me take a look at this function.

**Pull request merged**